src/Sdcb.PaddleOCR/PaddleOcrRecognizer.cs

```diff
@@ -174,6 +175,7 @@ private PaddleOcrRecognizerResult[] RunMulti(Mat[] srcs)
                             StringBuilder sb = new();
                             int lastIndex = 0;
                             float score = 0;
+                            var singleChars = new List<OcrRecognizerResultSingleChar>();
```

Comment:
I personally prefer not using var, instead, using C#'s latest gramma like: 
```csharp
List<OcrRecognizerResultSingleChar> singleChars = [];
```

or:
```csharp
OcrRecognizerResultSingleChar[] singleChars = [];
```